frontend/src/style.css

```diff
@@ -56,3 +56,19 @@ img {
   opacity: 0.2;
   pointer-events: none;
 }
+
+.inputStyle {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
```

Comment:
Biztos, hogy itt szeretnénk egy más font-family-t, mint az egész oldalon?

Reply:
Illetve ennyi fallback sem kell... Ez elég copy-paste szagú valahonnan.

Reply:
A radiobutton input stílusa eltért a többitől, ezért devToolsban megnéztem, hogy mik vannak a többire beállítva, és azt copy-pasteltem ide a radiobuttonnak. Gyakorlatilag így értem el, hogy egyformák legyenek.